src/function.test.ts

```diff
@@ -1054,3 +1054,20 @@ describe('m', () => {
     expect(compiled.parameters).toStrictEqual([]);
   });
 });
+
+describe('makePoint', () => {
+  test('Column argument', () => {
+    const query = db
+      .selectFrom('test')
+      .select((eb) =>
+        stf(eb).makePoint(-71.1043443253471, 42.3150676015829).as('alias'),
+      );
+    const compiled = query.compile();
+    expect(compiled.sql).toBe(
+      'select ST_MakePoint($1, $2) as "alias" from "test"',
+    );
+    expect(compiled.parameters).toStrictEqual([
+      -71.1043443253471, 42.3150676015829,
+    ]);
+  });
```

Comment:
We can add a test for the optional parameters. 
```suggestion

test('Column argument with optional parameters', () => {
    const query = db
      .selectFrom('test')
      .select((eb) =>
        stf(eb).makePoint(-71.1043443253471, 42.3150676015829, 1.42, 3,14).as('alias'),
      );
    const compiled = query.compile();
    expect(compiled.sql).toBe(
      'select ST_MakePoint($1, $2, $3, $4) as "alias" from "test"',
    );
    expect(compiled.parameters).toStrictEqual([
      -71.1043443253471, 42.3150676015829, 1.42, 3,14
    ]);
  });
  
  });
```